app/views/user/show.html.erb

```diff
@@ -1,3 +1,10 @@
-<h1>User#show</h1>
-<p>Find me in app/views/user/show.html.erb</p>
-<p>Harry Potter</p>
\ No newline at end of file
+<%= render partial: "partials/user", locals: { user: @user } %>
+<%= render partial: "partials/bio" %>
+<% @user.three_recent_posts.each do |post| %>
+  <%= render partial: "partials/post", locals:{ post: post } %>
+<% end %>
+<div class="button">
+  <button>
+    <%= link_to 'See all Post', user_post_path(@user) %>
+  </button>
+</div>
```

Comment:
- [ ] We need to implement the `comment and likes` features to be displayed in the view using this file.